.github/workflows/keep-alive.yml

```diff
@@ -7,11 +7,17 @@ jobs:
   keep-alive:
     name: Alive
     runs-on: ubuntu-latest
+    permissions:
+      actions: write
     steps:
       - uses: actions/checkout@v4
+<<<<<<< before updating
```

Comment:
This still contained a merge conflict @ivergara

Reply:
Oops. Thanks for noticing.